pkg/bootstrap/docker/openshift/servicecatalog.go

```diff
@@ -35,11 +35,37 @@ func (h *Helper) InstallServiceCatalog(f *clientcmd.Factory, publicMaster, catal
 		return errors.NewError("cannot obtain API clients").WithCause(err).WithDetails(h.OriginLog())
 	}
 
-	// Grant all users with the edit role, the ability to manage service catalog instance/binding resources.
+	// Grant all users with the view role, the ability to view service catalog instance/binding resources and
+	// podpreset resources.
+	scViewRule, err := authzapi.NewRule("get", "list", "watch").Groups("servicecatalog.k8s.io").Resources("instances", "bindings").Rule()
```

Comment:
`instances.servicecatalog.k8s.io` has a `spec.parameters` field which appears likely to hold secrets entered by the user.  `view` is a non-escalating role`, so granting `view` to a user shouldn't allow them to get at your secret information.